Command.fs

```diff
@@ -135,7 +135,7 @@ module Compose =
     ///     The next available intermediate stage number.
     ///     If the expression has no intermediate stages, we return 0.
     /// </returns>
-    and nextBoolIntermediate =
+    and nextBoolIntermediate : BoolExpr<Sym<_>> -> bigint =
```

Comment:
I think it just asks the type inferrer to kick in at that point.  Really `_` is evidence either of laziness on my part, or me thinking that it literally does not matter what the type is there.

There are some annotations where I've used `_` to mean 'this doesn't change', which isn't quite right.